code/modules/clothing/shoes/xeno/tajara.dm

```diff
@@ -0,0 +1,14 @@
+/obj/item/clothing/shoes/tajara
+	name = "native tajaran foot-wear"
+	desc = "Native foot and leg wear worn by Tajara, completely covering the legs in wraps and the feet in adhomian fabric."
+	icon = 'icons/obj/tajara_items.dmi'
+	icon_state = "adhomai_shoes"
+	item_state = "adhomai_shoes"
+	body_parts_covered = FEET|LEGS
+	species_restricted = list("Tajara")
+	contained_sprite = TRUE
+	description_fluff = "Today the fashion that dominates Adhomai shares few similarities to the clothing of old Furs, linen, hemp, silk and other such fabrics were traded for \
+	synthetic versions, creating a massive boom in the nylon industry and textile industry in the cities. Jeans, overcoats, army uniforms, parade uniforms, flags, pants, shirts, ties, \
+	suspenders, overalls are now the fashion of every Tajara from Nal'Tor to Kaltir. The protests of \"Old fashion\" supporters usually shatter against how effective and cheap it is to \
+	make the human clothes are. There are a few notable branches, the long-coat and fedora \"gangster style\". Leather jacket wearing \"Greaser\" or the popular amongst females, short \
```

Comment:
'The protests of "Old Fashion\" supporters usually can't stand against how undeniably effective and cheap to produce Human clothes are.'